collector/lib/CollectorConfig.cpp

```diff
@@ -175,6 +175,15 @@ void CollectorConfig::HandleAfterglowEnvVars() {
     afterglow_period_micros_ = static_cast<int64_t>(atof(afterglow_period) * 1000000);
   }
 
+  int64_t max_afterglow_period_micros = 300000000;  // 5 minutes
+
+  if (afterglow_period_micros_ > max_afterglow_period_micros) {
+    CLOG(WARNING) << "User set afterglow period of " << afterglow_period_micros_ / 1000000
+                  << " is greater than the maximum allowed afterglow period of " << max_afterglow_period_micros / 1000000;
+    CLOG(WARNING) << "Setting the afterglow period to " << max_afterglow_period_micros / 1000000;
```

Comment:
Could we maybe add time units to the message?

```suggestion
    CLOG(WARNING) << "User set afterglow period of " << afterglow_period_micros_ / 1000000
                  << "s is greater than the maximum allowed afterglow period of " << max_afterglow_period_micros / 1000000 << "s";
    CLOG(WARNING) << "Setting the afterglow period to " << max_afterglow_period_micros / 1000000 << "s";
```